File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.js
function solution(n) {
    let answer = 0;
    for (let i = 1; i <= 6 * n; i++) {
        if ((i * 6) % n === 0) {
            answer = i;
            break;
            // 피자의 조각 갯수를 인원수로 나눈 값이
            // 0이 될 때까지 for문을 실행하고, 만약
            // 0이 되면 for문을 빠져나간다.
        }
    }
    return answer;
}

File: 프로그래머스/lv0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        for(int i = 1; i <= 6 * n; i++) {
            if(i * 6 % n == 0) {
                answer = i;
                break;
                // 피자의 조각 갯수를 인원수로 나눈 값이 
                // 0이 될때 까지 for문을 실행하고 만약
                // 0이 되면 for문을 빠져나간다
            }
        }
        return answer;
    }
}